src/main.js

```diff
@@ -322,17 +322,29 @@ class MiniGraphCard extends LitElement {
     `;
   }
 
+  renderLoader() {
+    return html`
+        <div style="display: flex;">
+             <ha-circular-progress indeterminate style="margin: auto;"></ha-circular-progress>
```

Comment:
Can we please put the style part into the CSS section of the project?

Reply:
Agreed.

Reply:
Done